packages/playground/src/components/pipeline-list.tsx

```diff
@@ -69,3 +69,5 @@ export default function PipelineList({ ...props }: PageProps) {
     </>
   )
 }
+
+export { PipelineList, type Pipeline }
```

Comment:
let's keep `PipelineList` as the default export, and add a seprate type export for Pipeline

Reply:
Done